compiler/rustc_errors/src/tests.rs

```diff
@@ -27,10 +27,12 @@ fn make_dummy(ftl: &'static str) -> Dummy {
     let langid_en = langid!("en-US");
 
     #[cfg(parallel_compiler)]
-    let mut bundle = FluentBundle::new_concurrent(vec![langid_en]);
+    let mut bundle: FluentBundle =
+        IntoDyn(crate::fluent_bundle::bundle::FluentBundle::new_concurrent(vec![langid_en]));
 
     #[cfg(not(parallel_compiler))]
-    let mut bundle = FluentBundle::new(vec![langid_en]);
+    let mut bundle: FluentBundle =
+        IntoDyn(crate::fluent_bundle::bundle::FluentBundle::new(vec![langid_en]));
```

Comment:
ditto

Reply:
Yea, `FluentBundle` in the crate doesn't impl `new` function either